solana/bridge/program/tests/integration.rs

```diff
@@ -335,40 +435,6 @@ fn test_invalid_emitter(context: &mut Context) {
     .is_err());
 }
 
-fn test_duplicate_messages_fail(context: &mut Context) {
```

Comment:
Is there any particular reason why this test was dropped?  If it's ok for it to be removed I can also drop it from my refactor, which will let us run the tests in parallel.